package.json

```diff
@@ -40,12 +40,10 @@
   "dependencies": {
     "babel-plugin-htmlbars-inline-precompile": "0.0.1",
     "ember-cli-babel": "^5.0.0",
```

Comment:
This needs to be updated once ember-cli/ember-cli-htmlbars#53 is merged and a new release is cut.